src/app/upgrade.cpp

```diff
@@ -261,15 +366,15 @@ void handleChangedDefaults(const DefaultPreferencesMode mode)
         QVariant current;
     };
 
-    const QVector<DefaultValue> changedDefaults
+    const DefaultValue changedDefaults[] =
     {
         {QLatin1String {"BitTorrent/Session/QueueingSystemEnabled"}, true, false}
     };
 
-    SettingsStorage *settingsStorage {SettingsStorage::instance()};
-    for (auto it = changedDefaults.cbegin(); it != changedDefaults.cend(); ++it)
+    auto *settingsStorage = SettingsStorage::instance();
+    for (const DefaultValue &it : changedDefaults)
```

Comment:
Well, you could also change the variable name... Okay, I'll pretend I didn't notice 😀

Reply:
I thought about it as well but I couldn't find a name. Do you have any suggestion?

Reply:
Well, just call it by type name: `const DefaultValue &defaultValue`. Or `value`, `val`...

Reply:
Done.